modules/core/src/v2/coins/eth.ts

```diff
@@ -331,17 +333,21 @@ export class Eth extends BaseCoin {
   /**
    * Query Etherscan for the balance of an address
    * @param address {String} the ETH address
+   * @param apiKey {String} optional etherscan API key to authenticate the requests with
    * @param callback
    * @returns {BigNumber} address balance
    */
-  queryAddressBalance(address: string, callback?: NodeCallback<any>): Bluebird<any> {
+  queryAddressBalance(address: string, apiKey?: string, callback?: NodeCallback<any>): Bluebird<any> {
```

Comment:
This is a breaking change to the API. I would be surprised if there are customers using this, but is there a way to do this without changing the API?

Could you pass it as part of the options when you create the bitgo object instead? Then you can store it in the environment.

Reply:
For example, if someone is doing something like this, they will not be able to upgrade without updating their code.

```javascript
bitgo.coin('eth').queryAddressBalance('0x1234...', (err, res) => doStuffWithBalance(res));
```
I would like to drop callback params, but we should do it all at once in a single major version bump.

Reply:
> Could you pass it as part of the options when you create the bitgo object instead? Then you can store it in the environment.

I could do that, where should I plug that in?